.github/workflows/pylint.yml

```diff
@@ -0,0 +1,42 @@
+name: Pylint
+
+on: 
+  pull_request:
```

Comment:
What is the reason behind not adding the GitHub action on push? Wouldn't be better to have it on push too in order to fix whatever the lint is proposing?

Reply:
So in this configuration, we actually can't change it. You need to have the lint run only on a pull request, because or else the build will fail as it won't have a PR to push to